api_backend_server/tests/integration_tests/test_activities_apis.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import requests
+from collections import namedtuple
+
+@pytest.fixture(scope="module")
+def credentials():
+    Credentials = namedtuple("Credentials", "name category_id activity_start activity_end")
+    credentials = Credentials(
+        name="dummy_activity",
+        category_id=1,
+        activity_start="2020-04-30 09:34:35.414677",
+        activity_end="2020-04-30 11:34:35.414677"
+    )
+    return credentials
+
+
+def test_create_activity_should_succeed():
```

Comment:
You need to pass the fixture as an argument to the test function if you'd like to use it. At the moment, the test fails due to an AtrributeError (python's scope resolution is aware only of *credentials* function, that does not have the name, category_id, activity_start, activity_end attributes you reference)